### PR TITLE
Create questions.md for review

### PR DESCRIPTION
**Pull request by anajsana**
**Please do not merge until 2022-02-28**
This PR adds `questions.md` file so the community can provide feedback and add comments to the OSPO Survey 2022 questions

### CODE REVIEW

File: 2022/questions.md [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+
+## Part 1
+1. Where is your company or organization on its open source journey? (Check all that apply) 
+* Consuming open source code in products or services
+* Contributing to upstream open source projects
+* Influencing open source projects via leadership or maintainer roles
+* Initiating or releasing open source projects
+* Collaborating with peers across open source projects and/or foundations
+* Not involved in open source
+* Don’t know
```
Comment (celestehorgan) [inline 2022/questions.md:32]:
```suggestion
## Part 1

1. Where is your company or organization on its open source journey? (Check all that apply) 
    * Consuming open source code in products or services
    * Contributing to upstream open source projects
    * Influencing open source projects via leadership or maintainer roles
    * Initiating or releasing open source projects
    * Collaborating with peers across open source projects and/or foundations
    * Not involved in open source
    * Don’t know
```

helpful documentation fairy tip: if you don't format your lists like this sometimes you'll run into issues with formatting displaying incorrectly in GitHub pages <3

- Always a full line (carriage return) after any header and before any text directly below it
- tab in for any subitems of a list to properly display them as a sublist 
   - for example, this is tabbed in to the bullet point above

File: 2022/questions.md [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+* Getting engineering support and buy-in
+* Setting a budget and estimating program costs
+* Assessing or quantifying existing open source use and contribution
+* Resources required to perform license compliance
+* Finding commercial dependencies
+* Tool selection
+* Other (please specify)
+>Routing: go to Part 5a
+## Part 4
+### Part 4a: Do Not Have an Open Source Program
```
Comment (celestehorgan) [inline 2022/questions.md:278]:
```suggestion
## Part 4

### Part 4a: Do Not Have an Open Source Program

```

Similar - whenever there's headers, you want a full line between them or else markdown renderers sometimes get confused

File: 2022/questions.md [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+* Facebook
+* Google
+* IBM
+* Microsoft
+* Red Hat
+* SAP
+* Uber
+* Verizon
+* VMware
+33. Excluding the aforementioned companies, please nominate up to three technology or software companies that exemplify good open source community citizenship in terms of contributions, collaboration and leadership on projects and initiatives within the open source ecosystems?
```
Comment (celestehorgan) [inline 2022/questions.md:347]:
```suggestion

33. Excluding the aforementioned companies, please nominate up to three technology or software companies that exemplify good open source community citizenship in terms of contributions, collaboration and leadership on projects and initiatives within the open source ecosystems?
```

In really long lists where items have sub-lists (like this survey) once again, a full line between list items helps markdown render correctly <3

File: 2022/questions.md (hunk 1 above)
Comment (anajsana) [reply 2022/questions.md:32]: You are totally right @celestehorgan thanks for making this format review!

File: 2022/questions.md [hunk 4]
```diff
@@ -0,0 +1,535 @@
...
+42. How often does your average application development team release code into production?
+   * Hourly
+   * Daily
+   * Weekly
+   * Monthly
+   * Quarterly
+   * Annually
+   * Don’t know
+
+## Part 7: Licensing, Compliance and Repos
```
Comment (CsatariGergely) [inline 2022/questions.md:423]: https://github.com/todogroup/osposurvey/issues/101

File: 2022/questions.md [hunk 5]
```diff
@@ -0,0 +1,535 @@
...
+   * Tern
+   * Veracode
+   * WhiteHat Security
+   * WhiteSource
+   * We utilize a homegrown tool or methodology for open source compliance
+   * We do not utilize a tool or methodology for open source compliance
+   * Other (please specify)
+<!---question wording is different than last year --->
+
+46. Does the information security function within your organization access data from software composition analysis tools that are used for automated open source compliance?
```
Comment (CsatariGergely) [inline 2022/questions.md:466]: https://github.com/todogroup/osposurvey/issues/104

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: 2022/questions.md [hunk 6]
```diff
@@ -0,0 +1,535 @@
...
+<!---changed to "or risk management" --->
+
+11. What percentage of your open source program’s time is spent collaborating with the following departments?  (Total can equal more than 100%)
+>{< 20%        21-40%        41-60%        61-80%        > 80%    Don't know}
+   * Engineering 
+   * IT
+   * Legal
+   * Security
+   * Upstream open source projects
+   * Other (please specify)
```
Comment (anajsana) [inline 2022/questions.md:124]: add question https://github.com/todogroup/osposurvey/issues/106

**Comment by anajsana** [comment]
Questions and the new structure were implemented in the 2022 questionnaire. It can be found in the dedicated folder: https://github.com/todogroup/osposurvey/blob/main/2022/questions.md

**Pull request closed without merging**